src/crs.rs

```diff
@@ -14,7 +14,12 @@
 
 use std::collections::BTreeMap;
 
-use rustc_serialize::json::{self, ToJson};
+#[cfg(not(feature = "with_serde"))]
```

Comment:
Only because I didn't know you could? Are you saying using `serde` to serialize `rustc_serialize::json::Object` and `Value`s?